dashboard/templates/view_file_access_within_week.html

```diff
@@ -27,16 +27,14 @@ <h4 class="text-center">What Students Are Reading</h4>
     <input type="hidden" name="slider_num" id="slider_num" />
     <div/>
     <br/>
-    <h6 class="text-center">Files accessed by
+    <h6 class="text-center">Files accessed by students with grade
     <select class="grade" name="grade" id="grade">
         <option value="all" selected>All</option>
-        <option value="A" >A</option>
-        <option value="B" >B</option>
-        <option value="C" >C</option>
-        <option value="D" >D</option>
-        <option value="F" >F</option>
+        <option value=">=90" >90-100%</option>
```

Comment:
We probablay don't need > symbol in the text if you have the range here. Wouldn't be understood by all students anyway. Saying 90-100 seems the same thing.

Reply:
it is not in the text. But I have it fixed anyway -- "90-100" is used